src/containers/flows/MintWheels/steps/Info/Info.tsx

```diff
@@ -90,8 +90,8 @@ const Info = (props: InfoProps) => {
 			return (
 				<>
 					<p>
-						Each user may mint up to {props.maxPurchasesPerUser} Wheels. The
-						cost for each Wheels is <>{EthPerWheel} ETH</> plus GAS.
+						Each user may mint up to {props.maxPurchasesPerUser} Kicks. The cost
+						for each Kicks is <>{EthPerWheel} ETH</> plus GAS.
```

Comment:
Change `The cost for each Kicks is...` to `The cost for each pair of Kicks is...`